src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -23,19 +23,22 @@ package object csv {
    * Adds a method, `csvFile`, to SQLContext that allows reading CSV data.
    */
   implicit class CsvContext(sqlContext: SQLContext) {
-    def csvFile(filePath: String) = {
+    def csvFile(filePath: String,
```

Comment:
This implicit function is for the common case of of comma separated files (similar to tab separated files). That is why delimiter is hardcoded to ',' 
Similarly for the very common case we hard coded quote to '"'